networking/legacy-firewalls.md

```diff
@@ -58,15 +58,15 @@ Also, cluster administrators must have [configured IP pools]({{ site.baseurl }}/
 
 Annotate the pod with key `cni.projectcalico.org/ipv4pools` and/or `cni.projectcalico.org/ipv6pools` and value set to a list of IP pool names, enclosed in brackets.  For example:
 
-`cni.projectcalico.org/ipv4pools: "[\"pool-1\", \"pool-2\"]"`
+`cni.projectcalico.org/ipv4pools: "["pool-1", "pool-2"]"`
 
 Note the use of the escaped \" for the inner double quotes around the pool names.
 
 #### Restrict all pods within a namespace to use an IP address range
 
  Annotate the namespace with key `cni.projectcalico.org/ipv4pools` and/or `cni.projectcalico.org/ipv6pools` and value set to a list of IP pool names, enclosed in brackets.  For example:
 
-`cni.projectcalico.org/ipv4pools: "[\"pool-1\", \"pool-2\"]"`
+`cni.projectcalico.org/ipv4pools: "["pool-1", "pool-2"]"`
```

Comment:
This change is not correct - we still need the backticks here, or we need to use different quotation marks.

e.g.,

```
cni.projectcalico.org/ipv4pools: '["pool-1", "pool-2"]'
```

Reply:
Same for the above

Reply:
Fixed.